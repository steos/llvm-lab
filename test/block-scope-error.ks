
	void main()
	{
		int x = 0;
		if (true) {
			int x = 3;
			iprintln(x);
		}
	}
	
	native void iprintln(int);