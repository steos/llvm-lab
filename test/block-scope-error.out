symbol x is already declared on line 6 at offset 7
